api/docs/release.dox

```diff
@@ -179,6 +179,10 @@ changes:
    which is required for producing whole-trace results in the parallel mode of analyzer
    operation, and also release_interval_snapshot() which is used to release the
    analyzer framework's claim to the interval snapshot objects.
+ - ud2a and ud2b have been renamed to ud2 and ud1, respectively. The old constants
+   OP_ud2a and OP_ud2b, as well as the INSTR_CREATE_ud2a()/INSTR_CREATE_ud2b() macros,
```

Comment:
Add `#` to get doxygen links to the OP_ names: `#OP_ud2a`, etc.  Maybe should mention `#OP_ud1` to get a link there too.

Reply:
That generally means you have an error and either omitted a Doxygen comment for the target or didn't pass that file to Doxygen -- please double-check.